packages/dht/src/connection/websocket/WebsocketConnector.ts

```diff
@@ -183,33 +183,29 @@ export class WebsocketConnector {
         for (const reattempt of range(ENTRY_POINT_CONNECTION_ATTEMPTS)) {
             const entryPoint = sample(this.config.entrypoints)!
             try {
-                if (!this.websocketServer) {
-                    return noServerConnectivityResponse
+                if (!this.config.entrypoints || this.config.entrypoints.length === 0) {
```

Comment:
As `config.entrypoints` doesn't change during the for loop iteration, this block could be moved out of the for loop? I.e. would be checked once before entering the for loop.

(Could also rename `config.entrypoints` e.g. to `config.connectivityRequestTargets` in a separate PR as discussed yesterday. Maybe in a separate PR.)